cpp/src/io/parquet/experimental/hybrid_scan_helpers.cpp

```diff
@@ -137,4 +149,110 @@ void aggregate_reader_metadata::setup_page_index(cudf::host_span<uint8_t const>
   }
 }
 
+std::tuple<std::vector<input_column_info>,
+           std::vector<inline_column_buffer>,
+           std::vector<cudf::size_type>>
+aggregate_reader_metadata::select_payload_columns(
+  std::optional<std::vector<std::string>> const& payload_column_names,
+  std::optional<std::vector<std::string>> const& filter_column_names,
```

Comment:
what happens/ what is supposed to happen when a column in one of these vectors does not match any columns in the file?

Reply:
We just ignore any column name not in the file inside `aggregate_reader_metadata::select_columns`